apps/member-profile/app/shared/components/resource-form.tsx

```diff
@@ -63,18 +63,25 @@ export function ResourceAttachmentField({
 
   return (
     <Form.Field
-      description="Must be less than 20 MB and one of the following file types: PNG, JPG, or PDF."
+      description="Please choose the file you want to upload."
       error={error}
       label="Attachment"
       labelFor={name}
-      required={required}
+      required
     >
-      <input
-        accept="image/png, image/jpeg, .pdf"
+      <FileUploader
+        accept={['application/pdf', 'image/jpeg', 'image/png']}
         id={name}
         name={name}
-        required={required}
-        type="file"
+        required
+        {...(defaultValue && {
+          initialFile: {
+            id: defaultValue.id,
+            name: defaultValue.id,
+            size: 0,
```

Comment:
Unfortunately we're not storing the file size in the database as of right now, so we won't be able to display it when showing the initial file.